dist/common/scripts/scylla_util.py

```diff
@@ -99,6 +99,19 @@ def scyllabindir():
 def sysconfdir():
     return str(sysconfdir_p())
 
+def get_required_aio_slots():
+    return str(cpu_count() * 11026 + 65536)
+
+def configure_aio_slots():
+    aio_max_nr_value = "fs.aio-max-nr = " + get_required_aio_slots()
+    with open("/etc/sysctl.conf", "r+") as file:
+        for line in file:
+            if "fs.aio-max-nr" in line:
+                break
+        else:
+            file.write(aio_max_nr_value)
+            os.system('sysctl -p /etc/sysctl.conf')
+
```

Comment:
what if sysctl.conf already have the fs.aio-max-nr but with wrong value. in this case it stay wrong and scylla_io_setup could fail.
May better to check if "fs.aio-max-nr" is in file, then update it with counted correct value and if not then write to file?

Reply:
@aleksbykov I was thinking that we _don't_ want to override something user has already configured. How about the following compromise: let's add a logging line here telling the user that `fs.aio.max-nr` was not updated because it was already configured?